depobs/website/views.py

```diff
@@ -135,9 +138,46 @@ def show_package_changelog() -> Any:
     )
 
 
+@api.route("/histogram.png")
+def get_histogram() -> Any:
+
+    scores = models.get_statistics()
+    counts = scores["score_codes_histogram"]
+
+    assert isinstance(counts, dict)
+    letters = list()
+    for letter in counts:
+        for i in range(counts[letter]):
+            letters.append(letter)
```

Comment:
nit: add comment that this is a hack to get seaborn to work properly including the example data below